### PR TITLE
Added nb to explore ways to convert pdf to QA dataset

### PR DESCRIPTION
**Pull request by suppathak**
Related #4

### CODE REVIEW

**Comment by suppathak** [comment]
I do not have access to assign or add reviewers. 

cc'ing it here: @codificat , @Shreyanand !! Thanks

File: notebooks/pdf_data_converter.ipynb [hunk 1]
```diff
@@ -0,0 +1,264 @@
+{
```
Comment (Shreyanand) [inline notebooks/pdf_data_converter.ipynb:1]: <p>This is already pretty awesome, thanks Surya! I guess we can add some text cleaning here to further improve it. How are we getting the context here? Maybe we could remove the header, the footer, and all the questions from the text and the remaining string can be one big context paragraph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (suppathak) [reply notebooks/pdf_data_converter.ipynb:1]: Thanks @Shreyanand . Context in this case is the whole content of the page that the answer is from. I am working on cleaning it up.

Comment (Shreyanand) [reply notebooks/pdf_data_converter.ipynb:1]: <p>The context as page idea is good, we can remove questions and unnecessary text from the page and add that as context. </p>

Comment (Shreyanand) [reply notebooks/pdf_data_converter.ipynb:1]: <p><em>Can I define a custom domain and certificate for my applications?</em></p><p><em>Yes. See Configuring custom domains for applications for more information.</em></p><p> </p><p>I was just looking at an output (ID 70) and it seems it is splitting at the full stop...</p>

Comment (suppathak) [reply notebooks/pdf_data_converter.ipynb:1]: <p>Yes. I think we need to clean up a bit. Currently, it is at the basic step , e.g, question and answer were splitted by '?'. and Sentences were splitted by "." </p>

**Comment by suppathak** [comment]
> One question: is this specific to [the FAQ page](https://www.redhat.com/en/technologies/cloud-computing/openshift/aws/faq) only?
> 
> I am asking because I believe this page might be "one of its kind", and if that's the case it might be better to just copy&paste and apply some formatting.
> 
> It seems to me that there are a few things in that page that could over-complicate this automated task (multiline answers, links within the text) that might make this not worth it.
> 
> What do you think?
> 
> Don't get me wrong: it looks like a great start! (thanks!) I'm just wondering if it is worth pursuing, as I believe mostly all the data we will work with will not be in Q/A (FAQ) format

I do agree with your point. Since we are not only looking for Q&A docs, but instead all kinds of docs. Need to come up with more general way to solve this Q&A problem. 
Thanks!

**Comment by codificat** [comment]
By the way, I found a very similar FAQ here: https://www.rosaworkshop.io/rosa/14-faq/
For that one there is a [source file in GitHub](https://github.com/openshift-cs/rosaworkshop/blob/22a8bf7ad0137e27b721d4c5904f122a6c6098e8/rosa-workshop/rosa/14-faq.md).

**Comment by suppathak** [comment]
Closing this since we are not more dealing with extractive QA anymore. We already have the markdown files and validation dataset in the [data](https://github.com/redhat-et/foundation-models-for-documentation/tree/master/data) folder.

**Pull request closed without merging**